### DISCUSSION TITLE
Is it possible to run the aspire dashboard docker container from docker compose?

### DISCUSSION THREAD

**Discussion by funkysi1701** [microsoft/aspire#2593] [Q&A]
Do you know if it is possible to run the aspire dashboard from docker compose?

I was thinking something like this but I don't think I have the mapping of ports between containers correct

version: "3.4"

services:
  Api:
    image: myapidockerimage
    container_name: api
    ports:
      - 4317:4317
  Aspire:
    image: mcr.microsoft.com/dotnet/nightly/aspire-dashboard:8.0.0-preview.4
    container_name: aspire_dashboard
    ports:
      - 18888:18888
      - 4317:18889

**Comment by davidfowl** [comment]
Of course you can! What's the issue?

**Comment by funkysi1701** [comment]
Dashboard runs fine but it wasn't getting any data. I added the endpoint to send the open telemetry to and I have now got it working

.AddOtlpExporter(exporterOptions => exporterOptions.Endpoint = new Uri("http://aspire_dashboard:18889"));

**Reply by davidfowl** [reply]
The sounds like you fixed it. What did you configure before? Or did you not have any configuration?

**Reply by funkysi1701** [reply]
yeah no configuration so it was sending to localhost, which obviously won't work in a docker compose scenario
